src/components/input/_base.scss

```diff
@@ -28,9 +28,18 @@ $transitions-label-height: 1.2rem;
     cursor: pointer;
     z-index: $z-2;
 
+    &::placeholder {
+        color: transparent;
+        transition: color 250ms;
+    }
+
     &:focus {
         outline: none;
         cursor: text;
+
+        &::placeholder{
```

Comment:
Hmm i think having this style under an element focus selector is unreliable. Since insome of our use cases involve programmatically "focusing" an element by setting a react state. 

So the component's focus is not necessarily the same as the element focusing.